src/code/components/organism.js

```diff
@@ -1,6 +1,8 @@
 import React, {PropTypes} from 'react';
 
-const OrganismView = ({org, id, className="", width=200, flipped=false, style={}, onClick, wrapper }) => {
+const DEFAULT_DRAKE_HEIGHT = 200;
+
+const OrganismView = ({org, id, className="", height, width, flipped=false, style={}, onClick, wrapper }) => {
```

Comment:
it would be nice not to require that all organism images in the future are square, though.